### ISSUE TITLE
Extraction of individual Alpha blends from Lightmapped_4WayBlend textures

### ISSUE THREAD

**Issue by Herobrine24** [GorangeNinja/PyVMF#7]
Hi,
I was wondering if PyVMF is capable of extracting the individual Alpha Values from 4 blended textures using Lightmapped_4WayBlend and insert the Alpha Values for specifc textures of the blend into different displacements using WorldVertexTransition.

**Comment by surf-gorange** [comment]
I'm very confused about what you're trying to achieve here, what alpha values would you extract from the textures? Where in the displacement do you want to "insert" those alpha values?

PyVMF doesn't do .vmt and .vtf stuff, though it does allow you to change the displacement values.

**Comment by Herobrine24** [comment]
Im involved in porting Black Mesa maps to SFM and since SFM doesnt support the blending of 4 textures on displacements, these displacements have to be retextured. Right now were just using the dominant texture seen on the displacement however Im curious if PyVMF could extract the blend values of individual textures from the vmfs so that they can be applied to displacements with a texture with a supported material type.

TL;DR: can PyVMF extract the blend data of 4 textures individually, inside the vmfs and apply it to a different vmf?

**Comment by surf-gorange** [comment]
It's technically possible, but I haven't added support for 4 way textures yet so you would have to write it yourself.
